src/Components/Game.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import './Game.css'
+export default class Game extends React.Component {
+    state = {
+        score: 0,
+        points: 1
+    }
+    handleStart = () => {
+        this.setState({
+            score: 0,
+            points: 1
+        })
+    }
+
+    handlePoints = () => {
```

Comment:
preferably you want to update the state by using a callback since the state you want to update depends on the previous state.